DefaultSettings.php

```diff
@@ -1,26 +1,17 @@
 <?php
 
-namespace SMW;
-
 /**
- * SemanticMediaWiki default settings
+ * DO NOT EDIT!
```

Comment:
Well, I didn't want to touch this at all before 2.4 but now that my hand was forced I did it in one flush.